pkg/handlers/validation_builder.go

```diff
@@ -39,10 +40,10 @@ func MinLen(min int) ValidateOption {
 	}
 }
 
-func MaxLen(min int) ValidateOption {
+func MaxLen(max int) ValidateOption {
 	return func(field string, value *string) *errors.ServiceError {
-		if value != nil && len(*value) > min {
-			return errors.MinimumFieldLengthNotReached("%s is not valid. Maximum length %d is required.", field, min)
+		if value != nil && len(*value) > max {
+			return errors.MinimumFieldLengthNotReached("%s is not valid. Maximum length %d is required.", field, max)
```

Comment:
```suggestion
			return errors.MaximumFieldLengthMissing("%s is not valid. Maximum length %d is required.", field, max)
```

Not sure however `MaximumFieldLengthMissing` is a good name. I'd change that to `MaximumFieldLengthExceeded`

Reply:
thanks. fixed that too